src/main/cli/cli.c

```diff
@@ -6167,9 +6166,9 @@ static void cliDshotTelemetryInfo(const char *cmdName, char *cmdline)
 #endif
         for (uint8_t i = 0; i < getMotorCount(); i++) {
             cliPrintf("%5d   %7d   %6d   %5d   ", i,
-                      (int)getDshotTelemetry(i) * 100,
-                      (int)getDshotTelemetry(i) * 100 * 2 / motorConfig()->motorPoleCount,
-                      (int)getDshotTelemetry(i) * 100 * 2 / motorConfig()->motorPoleCount / 60);
+                      getDshotTelemetry(i) * 100,
+                      erpmToRpm(getDshotTelemetry(i)),
+                      erpmToRpm(getDshotTelemetry(i)) / 60);
```

Comment:
```
            const uint16_t dshot = getDshotTelemetry(i);
            const uint16_t rpm = erpmToRpm(getDshotTelemetry(i));
            cliPrintf("%5d   %7d   %6d   %5d   ", i,
                      dshot * 100,
                      rpm,
                      rpm / 60;
```

Can save more FASH and avoid inlining. Please double check that I haven't done mistake in rewriting your code.

Reply:
Reworked

Reply:
(I would expect that CSE is smart enough to eliminate extra code)